frontend/src/components/FacilityBooking/ViewBookingCard.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import ViewBookingCardButton from '../../assets/viewBookingCardButton.svg'
+import ViewBookingCardUserIcon from '../../assets/viewBookingUserIcon.svg'
+
+import { Booking } from '../../store/facilityBooking/types'
+import { unixToFullDateNumeric } from '../../common/unixToFullDateNumeric'
+import { unixTo12HourTime } from '../../common/unixTo12HourTime'
+import { unixToFullDay } from '../../common/unixToFullDay'
+import { DOMAIN_URL, ENDPOINTS } from '../../store/endpoints'
+import { openUserTelegram } from '../../common/telegramMethods'
+
+const BackgroundOverlay = styled.div`
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.7);
+  z-index: 202;
+  cursor: pointer;
+`
+
+const BookingContainer = styled.div`
+  position: absolute;
+  padding-bottom: 20px;
+  top: 80px;
+  gap: 10px;
+  width: 85%;
+  max-width: 400px;
+  cursor: pointer;
+  background-color: #ffffff;
+  border-radius: 20px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  z-index: 205;
+`
+
+const BookingHeader = styled.p`
+  width: 100%;
+  height: 20%;
+  cursor: pointer;
+  background-color: #f1f3f7;
+  border-radius: 20px 20px 0px 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  z-index: 2;
+  text-align: center;
+  padding: 20px 0px;
+`
+
+const SubContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+const EventDetailsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 15px;
+  align-items: center;
+`
+
+const EventTimingContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const ExitButtonContainer = styled.img`
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+`
+
+const TelegramButtonContainer = styled.img`
+  width: 39px;
+  height: 39px;
+  cursor: pointer;
+`
+
+const StyledText = styled.text<{ largeFont?: boolean; boldFont?: boolean; grayed?: boolean }>`
+  font-size: ${(props) => (props.largeFont ? `22px` : `14px`)};
+  color: ${(props) => (props.grayed ? `#666666` : `#191919`)};
+  font-weight: ${(props) => (props.boldFont ? `700` : `400`)};
+  text-align: center;
+  line-height: 22px;
+`
+
+type Props = {
+  booking?: Booking | null
```

Comment:
```suggestion
  booking?: Booking
```
no need to specify null